onadata/libs/utils/common_tools.py

```diff
@@ -104,3 +104,28 @@ def get_response_content(response):
         contents = response.content
 
     return contents
+
+
+def json_stream(data, json_string):
+    """
+    Generator function to stream JSON data
+    """
+
```

Comment:
remove blank line, other functions do not have a line between docstring and code